src/CartServiceProvider.php

```diff
@@ -6,7 +6,13 @@
 use GuzzleHttp\Client;
 use Illuminate\Support\ServiceProvider;
 use Lab19\Cart\Services\CartService;
+use Lab19\Cart\Services\CurrencyConversionInterface;
+use Lab19\Cart\Services\GeolocationInterface;
+use Lab19\Cart\Services\GeoLocationService;
```

Comment:
@AlexeiaW let's make the GeoLocationService the same as the interface in terms of camelcase, so GeolocationService (no upper case L).

Reply:
Renamed the service